src/bin/dual-iir.rs

```diff
@@ -425,26 +457,118 @@ mod app {
             }
         }
 
+        // Update Pounder configurations
```

Comment:
This should also go into `PounderSettings` and/or `Pounder`. Don't spread it into every application.